modules/tinymce/src/core/main/ts/api/EventTypes.ts

```diff
@@ -101,6 +101,7 @@ export interface EditorEventMap extends Omit<NativeEventMap, 'blur' | 'focus'> {
   'ProgressState': ProgressStateEvent;
   'AfterProgressState': AfterProgressStateEvent;
   'PlaceholderToggle': PlaceholderToggleEvent;
+  'ViewUpdate': { };
```

Comment:
```suggestion
```
As mentioned in my previous comment, this isn't part of the current core so we shouldn't have types, etc... if we were to add this then it should also have an implementation.

Reply:
I was half expecting to do the implementation now, and have word count listen for _only_ `ViewUpdate`, but maybe that should wait for Tiny 6.

Reply:
I guess if we hide it now and don't expose it or document it we can more easily change it if needed then we more the alt model into core. I thought it would be nice to expose it since it's now used by a core plugin.

Reply:
done